### ISSUE TITLE
Date-time picker 508 compliance

### ISSUE THREAD

**Issue by obr42** [beer-garden/react-ui#304]
This applies anywhere the Date-time picker is used. For some reason labels are flagged as wrong/missing. Adding `aria-label` or `aria-labelledby` adds "Elements must only use allowed ARIA attributes" flag but doesn't remove the "Form elements must have labels" flag. Can remove flags by adding `label` (and `InputLabelProps={{shrink: true}}` to make the label stay superscript-ed) but can't find a way to then hide the label and it is incompatible with existing ticket to remove extra space around the filter inputs to have the label showing. Adding `InputLabelProps={{'aria-label': 'Start date'}}` did not remove the flag either.
